src/test/java/wellnus/tokenize/ReflectionTokenizerTest.java

```diff
@@ -0,0 +1,54 @@
+package wellnus.tokenize;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import wellnus.exception.TokenizerException;
+import wellnus.storage.ReflectionTokenizer;
+
+public class ReflectionTokenizerTest {
+    private static final int INDEX_ZERO = 0;
+    private static final int INDEX_ONE = 1;
+    @Test
+    void tokenizeReflect_checkOutput_success() throws TokenizerException {
+        ArrayList<Set<Integer>> indexToTokenize = new ArrayList<>();
```

Comment:
I recommend something like `indexesToTokenize` instead, this is a plural data structure